src/pages/Replay.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { withRouter, RouteComponentProps } from "react-router-dom";
+
+import Header from "../components/Header";
+import { Button, Typography } from "antd";
+import "../styles/Replay.css";
+
+const Replay: React.FC<RouteComponentProps> = (props) => {
+  const playAgainForSameCourse = () => {
+    // TODO: Need to pass in the course/game ID to replay for the same course (or is it stored in Redux/Context API?)
+    props.history.push("/DisplayCards");
+  };
+
+  const playAgainForDifferentCourse = () => {
+    // TODO: Change URL here for course information page
+    // props.history.push("/DisplayCards");
```

Comment:
I think this can be directed to the CourseInfo page now

Reply:
@mitchfaulcon I'm going to do all the linking things in their own ticket, honestly just waiting for all the authentication stuff to be sorted out since I'll have to pass certain parameters back into the page if the game is replayed.